cdap-watchdog/src/main/java/co/cask/cdap/logging/appender/file/FileLogAppender.java

```diff
@@ -82,13 +84,14 @@ public FileLogAppender(CConfiguration cConfig,
                          DatasetFramework dsFramework,
                          TransactionExecutorFactory txExecutorFactory,
                          NamespacedLocationFactory namespacedLocationFactory,
-                         RootLocationFactory rootLocationFactory) {
+                         RootLocationFactory rootLocationFactory, Impersonator impersonator) {
```

Comment:
My understanding is that FileLogAppender is only used in Standalone (not in distributed).
Impersonation is not supported in Standalone.